panel/command/__init__.py

```diff
@@ -91,7 +91,7 @@ def main(args: list[str] | None = None):
             try:
                 ret = parsed_args.invoke(parsed_args)
             except Exception as e:
-                if config.autoreload:
+                if config.autoreload or config.log_level in ("ERROR", "CRITICAL"):
```

Comment:
That seems the exact opposite of what we want, i.e. we want to show the traceback for more verbose log levels:

```suggestion
                if config.autoreload or config.log_level in ("DEBUG", "INFO"):
```